### ISSUE TITLE
a11y: Project view

### ISSUE THREAD

**Issue by woodruffw** [pypa/warehouse#6012]
Accessibility improvements can be made to `/project/{project}/`:

In the history tab:

* `<div class="release__graphic">`  has child `<div>`s that could either be `aria-hidden=true` **or** should have descriptive `alt` texts (current text is "History Node" for both current and past releases)

In the files tab:

* The file `<table>` itself should have a `<caption>`, as the first child node. This is partially mitigated by the `<h2>` above
* The  `<table>` should include `scope=col` or another appropriate scope for each `<th>` node

**Comment by woodruffw** [comment]
More improvements:

* The `pip` installation suggestions lack an `aria-label`
* The button to copy the `pip` invocation into the clipboard lacks `type=button`

Screenshot:

<img width="433" alt="Screen Shot 2019-06-17 at 1 52 20 PM" src="https://user-images.githubusercontent.com/3059210/59625383-24d13600-9107-11e9-804f-21e1d9125622.png">

**Comment by woodruffw** [comment]
More improvements:

* For the maintainers section (see screenshot 1), the user icon and username links can be combined into a single link, avoiding content duplication with a screen reader.

* For the "Last released" element (see screenshot 2), use Jinja2's `localize` instead of `humanize` to insert proper ARIA tags.

<img width="342" alt="Screen Shot 2019-06-18 at 10 55 15 AM" src="https://user-images.githubusercontent.com/3059210/59695056-9668bd00-91b7-11e9-965f-a2ba638daf99.png">

<img width="269" alt="Screen Shot 2019-06-18 at 11 01 48 AM" src="https://user-images.githubusercontent.com/3059210/59695598-7c7baa00-91b8-11e9-8156-6b5ad12e865a.png">
